tests/main/command-chain/task.yaml

```diff
@@ -28,6 +33,11 @@ prepare: |
     fi
 
 execute: |
+    # No needed to test the snap pkg when it is coming from the repository
+    if [ "$SNAP_REEXEC" = "0" ] && tests.info is-snapd-pkg-repo; then
```

Comment:
I don't get why there's no need to test this. Is the test failing?

Reply:
there is a bug in the deb which makes snapd fail to pack snaps.

Reply:
also I don't see any value running those tests with reexec=0 when we use snapd deb from the repo.